client/blocks/importer/wordpress/import-everything/index.tsx

```diff
@@ -212,12 +213,10 @@ export class ImportEverything extends SectionMigrate {
 
 	renderMigrationComplete() {
 		const { isMigrateFromWp } = this.props;
-		return (
-			<Hooray>
-				{ ! isMigrateFromWp
-					? this.renderDefaultHoorayScreen()
-					: this.renderHoorayScreenWithDomainInfo() }
-			</Hooray>
+		return ! isMigrateFromWp ? (
+			this.renderMigrationSuccessScreen()
+		) : (
+			<Hooray>{ this.renderHoorayScreenWithDomainInfo() }</Hooray>
```

Comment:
Now, we have only one Hooray screen.
Can we move `Hooray` wrapper to the `renderHoorayScreenWithDomainInfo` method.

So we can have something like:
```
return ! isMigrateFromWp
  ? this.renderMigrationSuccessScreen()
  : this.renderHoorayScreenWithDomainInfo()
```

Reply:
Great! Will address your suggestion 👍